src/pages/[id]/index.js

```diff
@@ -50,11 +43,21 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
     fillActiveTasksArray();
   }, [fillActiveTasksArray]);
 
+  if (errorMessage) {
+    return <NotFound errorMsg={errorMessage} />;
+  }
+
+  if (!isSuperUser && !superUserVerified) {
```

Comment:
`superUserVerified` checks for whether the call was made or not? From the name, it seems like it verifies if the current user is indeed a super user.

Reply:
yeah, the name `superUserVerified ` was confusing. renamed it to `userApiCalled`